recipes-bsp/u-boot/u-boot-envs-atmel.inc

```diff
@@ -1,16 +1,20 @@
 # U-Boot Environments for AT91 boards
 
-SRC_URI_append = " file://*.txt"
+SRC_URI_append = " file://envs/"
 
 ENV_SIZE = "0x4000"
 ENV_FILENAME = "uboot.env"
 ENV_FILEPATH = "${WORKDIR}/envs"
 
-do_deploy_append() {
+do_compile_append() {
     if [ -e "${ENV_FILEPATH}/${MACHINE}.txt" ]; then
-        echo "Compile u-boot environment for ${MACHINE} and deploy it."
+        echo "Compile U-Boot environment for ${MACHINE} and deploy it."
```

Comment:
I think that with the rename of the function, the deploy part of the comment can be dropped.

Reply:
fixed ;)